README.md

```diff
@@ -1,32 +1,85 @@
 [![codecov](https://codecov.io/github/redis/redis/graph/badge.svg?token=6bVHb5fRuz)](https://codecov.io/github/redis/redis)
 
-This README is just a fast *quick start* document. You can find more detailed documentation at [redis.io](https://redis.io).
-
-### What is Redis?
+Redis is an open-source, in-memory data structure store. People use Redis as a database, cache, message broker, streaming engine, and vector store. This document serves as both a quick start guide to Redis and a detailed resource for building it from source.
```

Comment:
I suggest removing this sentence because we have the "What is Redis" section below, and because this is just a partial description of what is there.

Reply:
Sure